src/vs/workbench/contrib/inlineChat/browser/inlineChatController.ts

```diff
@@ -214,6 +214,11 @@ export class InlineChatController implements IEditorContribution {
 	}
 
 	private async [State.CREATE_SESSION](options: InlineChatRunOptions): Promise<State.CANCEL | State.INIT_UI | State.PAUSE> {
+		if (this._activeSession) {
```

Comment:
Adding this makes sense but we should first understand (and maybe fix) why this doesn't work:

https://github.com/microsoft/vscode/blob/acf1af0b053a560e9c5460746e1bc17eb9289cb8/src/vs/workbench/contrib/inlineChat/browser/inlineChatController.ts#L178

In theory, starting a new session should finish an existing session but it seems things aren't awaited property